.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Update all providers",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "program": "cmd/gluetun/main.go",
+            "args": [
+                "update",
+                "-all",
```

Comment:
I actually have a separate, uncommitted launch entry for SlickVPN! Agreed on the `-all` bit though.

Not sure about being able to prompt - but I bet it could pick it up from an environment variable or similar?

Reply:
Updated this with a launch config that asks the user for string input for provider(s) on run